salt/modules/openstack_mng.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+'''
+Module for OpenStack Management
+
+:codeauthor:    Konrad Mosoń <mosonkonrad@gmail.com>
+:maturity:      new
+:depends:       openstack-utils
+:platform:      linux
+'''
+
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+
+# Import salt libs
+import salt.utils
+
+# Import third party libs
+import os.path
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'openstack_mng'
+
+
+def __virtual__():
+    '''
+    Only load this module if openstack-service is installed
+    '''
+    if os.path.isfile('/usr/bin/openstack-service'):
+        return __virtualname__
+    else:
+        return (False, 'The openstack-service binary could not be found.')
+
+
+def start_service(service_name):
+    '''
+    Start OpenStack service immiedately
+
+    CLI Example:
+
+    .. code-block:: bash
+        salt '*' openstack_mng.start_service neutron
+    '''
+
+    if __salt__['cmd.retcode'](['/usr/bin/openstack-service', 'start', service_name]) == 0:
```

Comment:
It would be cleaner and shorter if we changed this logic in the functions that you've written to something more like this:

`return __salt__['cmd.retcode'](['/usr/bin/openstack-service', 'start', service_name]) == 0`